Test/docker-compose.yml

```
@@ -61,14 +61,38 @@ services:
       target: dep-runner
     env_file:
       - ../Config/server.sample.env
+    # expose:
+    #   - "54000"
     ports:
-      - "50020:54000"
+      - "54000:54000"
     volumes:
       - type: bind
         source: ./../Config
         target: /QuickMPC-BTS/Config
+    # depends_on:
+    #   bts_envoy:
+    #     condition: service_started
+    networks:
+      shared-network:
+        ipv4_address: 10.0.4.1
     command: [ "./BeaverTripleService" ]
 
+  bts_envoy:
+    tty: true # 起動しっぱなし
+    image: envoyproxy/envoy:v1.17.1
+    volumes:
+      - type: bind
+        source: ../Config/envoy.yaml
+        target: /etc/envoy/envoy.yaml
+    expose:
+      - "9901"
+    ports:
+      - "0.0.0.0:50020:50020"
```

It seems difficult because we can't switch whether to use envoy or not... 　
Also, I think it's too big to comment out.　　
How about allowing both in this `docker-compose.yaml` file?

```suggestion
    ports:
      - "0.0.0.0:51020:50020"
```

---

@arukuka
Do you mean that bts_envoy is activated whether you use it or not?

---

@KotaTakahashi9320 :bow:  
Yes, I thought it would be better.  
If we would like to pass envoy proxy, we could do it by accessing `51020` port with `WithEnvoy: true` configuration,  
otherwise, `50020` port could be used.

---

I will resolve this comment thread